.pylintrc

```diff
@@ -12,4 +12,7 @@ disable=
     unnecessary-pass,
     wrong-import-position,
     logging-format-interpolation,
-    too-many-lines
+    too-many-lines,
+    C0103,
```

Comment:
can you change these to descriptive names?

Reply:
No, those are the ones I found for the errors to increase the score.  If you find more descriptive ones, you are welcome to change them.

Reply:
Can you check if these names work:
C0103 => invalid-name
R0201 => no-self-use

You could also add:
too-few-public-methods
too-many-public-methods
unused-wildcard-import

Reply:
No, like I said, you are welcome to change them and try as you consider necessary.